api/bin/chainflip-broker-api/src/main.rs

```diff
@@ -309,6 +315,14 @@ impl RpcServer for RpcServerImpl {
 			.cf_get_affiliates(self.api.state_chain_client.account_id(), None)
 			.await?)
 	}
+
+	async fn affiliate_withdrawal_request(
+		&self,
+		affiliate_short_id: AffiliateShortId,
+	) -> RpcResult<()> {
```

Comment:
It would be good to return some useful information here, for example `WithdrawFeesDetail` like in a normal broker withdrawal.